lib/compress.js

```diff
@@ -3028,63 +3028,49 @@ merge(Compressor.prototype, {
         return this;
     });
 
-    var commutativeOperators = makePredicate("== === != !== * & | ^");
+    var reversibleOperators = makePredicate("+ * & | ^ < > <= >= == != === !==");
```

Comment:
> There is extra logic for + down here

I'm sorry, but that's too cryptic. '+' is not a reversible operation and doesn't belong in the set. Also, can we keep the name `commutativeOperators` as it has an unambiguous meaning?